Tasks/KubernetesV1/package.json

```diff
@@ -8,7 +8,7 @@
     "@types/uuid": "^8.3.0",
     "agent-base": "^6.0.2",
     "azure-pipelines-task-lib": "^4.0.1-preview",
-    "azure-pipelines-tasks-azure-arm-rest": "3.223.4",
+    "azure-pipelines-tasks-azure-arm-rest": "^3.226.0",
```

Comment:
Why is the version bump of the library needed?

Reply:
Hi @tkasparek , as a good practice while creating a PR, most of the time I execute 'npm audit fix' (which resolves CG vulnerabilities without breaking any changes), here the npm audit bumped this library version. I believe these library version change won't cause any issues and is certainly not the reason for the below CI check failure.